src/containers/course-section/CourseSectionContainer.tsx

```diff
@@ -37,22 +60,70 @@ const CourseSectionContainer: FC<SectionProps> = ({
 }) => {
   const { t } = useTranslation()
   const { openMenu, renderMenu, closeMenu } = useMenu()
+  const { openModal } = useModalContext()
 
   const [activeMenu, setActiveMenu] = useState<string>('')
   const [isVisible, setIsVisible] = useState<boolean>(true)
   const [titleInput, setTitleInput] = useState<string>(sectionData.title)
   const [descriptionInput, setDescriptionInput] = useState(
     sectionData.description
   )
+  const [lessons, setLessons] = useState<Lesson[]>([])
+  const [quizzes, setQuizzes] = useState<Quiz[]>([])
+  const [attachments, setAttachments] = useState<Attachment[]>([])
+  const [resources, setResources] = useState<CourseResources[]>([])
+  const [itemToDelete, setItemToDelete] = useState<CourseResources | null>(null)
+
+  useEffect(() => {
+    setResources((prevResources) => {
+      const allResourcesItems = [...lessons, ...quizzes, ...attachments]
+      const updatedResourcesItems = prevResources.filter((item1) =>
+        allResourcesItems.some((item2) => item2._id === item1._id)
```

Comment:
Maybe it's better to rename `item1`, `item2` to make code more understandable?

Reply:
renamed